mscore/prefsdialog.ui

```diff
@@ -3435,6 +3435,26 @@ Adjusting latency can help synchronize your MIDI hardware with MuseScore's inter
               <string>64th</string>
              </property>
             </item>
+            <item>
+             <property name="text">
+              <string>128th</string>
+             </property>
+            </item>
+            <item>
+             <property name="text">
+              <string>256th</string>
+             </property>
+            </item>
+            <item>
+             <property name="text">
+              <string>512th</string>
+             </property>
+            </item>
+            <item>
+             <property name="text">
+              <string>1024th</string>
+             </property>
+            </item>
```

Comment:
Not sure we need such short quantization values

Reply:
yes please, it is always useful when the tempo is very slow.